lib/options.h

```diff
@@ -28,7 +28,7 @@ limitations under the License.
 #include "error.h"
 #include "map.h"
 
-namespace Util {
```

Comment:
Nit for the future: we likely may eliminate `Util` namespace...

Reply:
> Nit for the future: we likely may eliminate `Util` namespace...

The Util namespace was added for things that aren't really specific to P4 -- just general utilities that weren't available is `std`, and we were trying to avoid dependencies on external things like boost, because they're a pain to deal with.